src/App.scss

```diff
@@ -1,39 +1,126 @@
-iframe {
-  display: none;
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;900&display=swap");
+
+body {
+  margin: 0;
+  width: 100vw;
+  height: 100vh;
+  background: #ecf0f3;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  place-items: center;
+  overflow: hidden;
+  font-family: poppins, sans-serif;
+}
+
+.App {
+  position: relative;
+  width: 550px;
+  border-radius: 20px;
+  padding: 40px;
+  box-sizing: border-box;
+  background: #ecf0f3;
+  box-shadow: 14px 14px 20px #cbced1, -14px -14px 20px white;
+}
+
+h1 {
+  margin-top: 10px;
+  font-weight: 900;
+  font-size: 1.8rem;
+  color: #1da1f2;
+  letter-spacing: 1px;
 }
 
-html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+.field {
+  text-align: center;
+  margin-top: 30px;
 }
 
 label,
+input,
+button,
+select {
+  display: block;
+  padding: 0;
+  width: 100%;
+  border: none;
+  outline: none;
+  box-sizing: border-box;
+  cursor: pointer;
+}
+
+label {
+  margin-bottom: 4px;
+}
+
+input::placeholder {
+  color: gray;
+}
+
+input,
+select {
+  background: #ecf0f3;
+  padding: 10px;
+  padding-left: 20px;
+  height: 50px;
+  font-size: 14px;
+  border-radius: 50px;
+  box-shadow: inset 6px 6px 6px #cbced1, inset -6px -6px 6px white;
+}
+
+select {
+  color: #789;
+}
+
+input {
+  caret-color: #b0c4de;
+}
+
 button {
+  color: white;
+  margin-top: 20px;
```

Comment:
It's better to style elements using a class selector, not tag. (you can leave it, if extra classes fail tests)
Also, please pay attention to CSS properties order.
Check yourself everywhere with these rules:
https://9elements.com/css-rule-order/

Reply:
Thank you, I changed some tags to class selectors and changed the order of CSS properties according to the rules in the link.